CMakeLists.txt

```diff
@@ -669,7 +669,9 @@ include(GNUInstallDirs)
 install(DIRECTORY ${ONNX_ROOT}/onnx
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
         FILES_MATCHING
-        PATTERN "*.h")
+        PATTERN "*.h"
+        PATTERN "backend/test/case" EXCLUDE
+        PATTERN "backend/test/data" EXCLUDE)
```

Comment:
tiny nit: it would be helpful if we can have a comment about why we exclude it for future reference.